presto-docs/src/main/sphinx/sql/set-session-authorization.rst

```diff
@@ -0,0 +1,29 @@
+=========================
+SET SESSION AUTHORIZATION
+=========================
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    SET SESSION AUTHORIZATION username
+
+Description
+-----------
+
+Changes the current user for all future security checks.
```

Comment:
Maybe we should just say "Changes the user of the current session" or "Changes the current user of the session"